src/comments.js

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable no-await-in-loop */
+
+const baseURL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps';
+
+const appID = '8a0dfW0tu0UAP5mKoeUq';
+
+export const fetchComments = async (showId) => {
+  const response = await fetch(`${baseURL}/${appID}/comments?item_id=${showId}`);
+  const data = response.ok ? await response.json() : [];
+  return data;
+};
+
+const saveComment = async (showId, commentData) => {
+  const data = {
+    item_id: showId,
+    username: commentData.name,
+    comment: commentData.comment,
+  };
+  const response = await fetch(`${baseURL}/${appID}/comments`, {
+    method: 'POST',
+    body: JSON.stringify(data),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  return response.ok;
+};
+
+export const updateShowWithComments = async (shows) => {
+  let i = 0;
+  while (i < 24) {
+    const showComments = await fetchComments(shows[i].id);
+    shows[i].comments = showComments;
+    i += 1;
+  }
+  return shows;
+};
+
+export const countComments = (show) => show.comments.length;
+
+const displayComment = (comment) => {
+  const commentDiv = document.createElement('div');
+  commentDiv.classList.add('comment');
+
+  const commentHeaderDiv = document.createElement('div');
+  commentHeaderDiv.classList.add('comment-header');
+  const author = document.createElement('h4');
+  author.innerText = comment.username;
+  const timestamp = document.createElement('small');
+  timestamp.innerText = comment.creation_date;
+  commentHeaderDiv.appendChild(author);
+  commentHeaderDiv.appendChild(timestamp);
+
+  const commentContent = document.createElement('p');
+  commentContent.innerText = comment.comment;
+
+  commentDiv.appendChild(commentHeaderDiv);
+  commentDiv.appendChild(commentContent);
+
```

Comment:
- [ ] I am able to see the comments which just submitted, but I'm not getting the last comment length. As you can see in the below screenshot, I have submitted one comment but still, the comments count to 0. Please update the comment count in real-time.

![Screenshot from 2021-12-24 17-39-45](https://user-images.githubusercontent.com/22434660/147353744-360c6c5f-404a-49af-a649-6def30d7681d.png)

Reply:
OK, done